### DISCUSSION TITLE
osu!Lazer multiplayer not allowing to play with friends

### DISCUSSION THREAD

**Discussion by TinyPeen69** [ppy/osu#19628] [Q&A]
### Type

Game behaviour

### Bug description

I tried to play osu!Lazer with a friend today but there was no ways to invite him and when we tried to look up each others lobbies by name nothing showed up.. ( and yes I did press "Create")

### Version

osu! 2022.731.1 - lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/9277379/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9277380/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9277381/updater.log)
[database.log](https://github.com/ppy/osu/files/9277382/database.log)
[network.log](https://github.com/ppy/osu/files/9277383/network.log)

**Comment by peppy** [comment]
Did you show password protected rooms from the filter control? You may have filtered the room away.

**Comment by naipofo** [comment]
Also make sure you have the same game mode selected (like osu!taiko or standard)
